### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by CarbonAlabel**
Disabled reactions to mentions of CREST and the XML API, fixed some commands, added two new commands to quickly get server status, and gave improving the status command another shot, as my last one didn't leave it looking all that pretty.

### CODE REVIEW

File: esi_bot/request.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 
 @command(trigger=re.compile(
-    r"^(https://(esi\.evetech\.net|esi\.tech\.ccp\.is))?/(?P<esi_path>.+)$"
+    r"^<?(https://esi\.(evetech\.net|tech\.ccp\.is))?/(?P<esi_path>.+?)>?$"
```
[the remark this answers is not in the document]
Comment (CarbonAlabel) [reply esi_bot/request.py]: It disables greedy matching, without it the closing bracket gets swallowed by the path.

[the remark this answers is not in the document]
Comment (a-tal) [reply esi_bot/request.py]: right-o. cheers

File: esi_bot/processor.py [hunk 2]
```diff
@@ -40,8 +40,8 @@
 )
 
 REACTION_TRIGGERS = {
-    re.compile(r"(^|.*( |!))crest( |$|\!|\?|\.)", re.IGNORECASE): "rip",
-    re.compile(r"(^|.*( |!))xml(api)?( |$|\!|\?|\.)", re.IGNORECASE): "wreck",
+    # re.compile(r"(^|.*( |!))crest( |$|\!|\?|\.)", re.IGNORECASE): "rip",
```
Comment (a-tal) [inline esi_bot/processor.py]: this was voted down

https://www.strawpoll.me/15876652/r

lets leave it in for now, not really doing any harm as is

Comment (CarbonAlabel) [reply esi_bot/processor.py]: @a-tal undone.

**Pull request merged**